packages/wxt/src/core/builders/vite/index.ts

```diff
@@ -71,7 +73,12 @@ export async function createViteBuilder(
       wxtPlugins.wxtPluginLoader(wxtConfig),
       wxtPlugins.resolveAppConfig(wxtConfig),
     );
-    if (wxtConfig.analysis.enabled) {
+    if (
+      wxtConfig.analysis.enabled &&
+      // If included, vite-node entrypoint loader will increment the
+      // bundleAnalysis's internal build index tracker, which we don't want
+      !baseConfigOptions?.excludeAnalysisPlugin
+    ) {
```

Comment:
I also fixed an issue with E2E tests where the analytics part files containing each build step's bundle info did not start at 0 because the `vite-node` imports were incrementing it's internal counter. Fixed by excluding this plugin when importing entrypoints.